src/panels/Menu/TaskBar/TaskBar.tsx

```diff
@@ -61,9 +61,13 @@ export function TaskBar() {
               }}
             >
               {item?.renderMenuButton ? (
-                item.renderMenuButton(eventHandlers(item))
+                item.renderMenuButton(eventHandlers(item), itemConfig)
               ) : (
-                <TaskBarMenuButton {...eventHandlers(item)} aria-label={item.title}>
+                <TaskBarMenuButton
+                  {...eventHandlers(item)}
+                  isOpen={itemConfig.isOpen}
+                  aria-label={item.title}
+                >
```

Comment:
This here is what I'm referring to in my other comment. We could just pass in item.id here, and do all the work in the `TaskBarMenuButton`. Or am I missing something? We could even place the aria-label in there which would mean we only need to pass 1 prop. That would be neat!